lib/ash_json_api/json_schema/json_schema.ex

```diff
@@ -594,8 +600,14 @@ defmodule AshJsonApi.JsonSchema do
     }
   end
 
-  defp route_in_schema(%{type: type, relationship: relationship}, _api, resource)
+  defp route_in_schema(
+         %{type: type, relationship: relationship, action: action, action_type: action_type},
+         _api,
+         resource
+       )
        when type in [:post_to_relationship, :patch_relationship, :delete_from_relationship] do
+    accept = Ash.Resource.action(resource, action, action_type).accept
```

Comment:
We won't want to use `accept` here actually. This is for adding/removing related entities. We can leave this one the way it is for now.

Reply:
will revert it then.